br/pkg/lightning/backend/external/reader.go

```diff
@@ -63,6 +63,8 @@ func readAllData(
 		task.End(zap.ErrorLevel, err)
 	}()
 
+	output.size = 0
```

Comment:
better to move to ⬇️
https://github.com/pingcap/tidb/blob/bd17acde8cc3684a349d208ff8f4ee2ac6fef1ba/br/pkg/lightning/backend/external/engine.go#L326-L329

resetting these fields in one place